src/server/config.ts

```diff
@@ -203,6 +204,14 @@ export const ApiConfig: ApiDataRequestConfig = {
     postponeFetch: !FeatureToggle.vergunningenActive,
     passthroughOIDCToken: true,
   },
+  POWERBROWSER: {
+    method: 'POST',
```

Comment:
why uppercase?

Reply:
Because it's uppercase in all the other ones as well.